src/e2e/puppeteer/helpers/screenshot-with-no-antialiasing.ts

```diff
@@ -0,0 +1,63 @@
+import { ScreenshotOptions } from 'puppeteer'
+import { page } from '../setup'
+
+/** Takes a screenshot with antialiasing disabled. */
+const screenshot = async (options?: ScreenshotOptions) => {
```

Comment:
Can be refactored in a separate PR, but do you think we could apply this as the default? i.e. Is there ever a reason to take an antialiased screenshot?